src/tests/test_state_manager.cpp

```diff
@@ -173,16 +174,16 @@ bool test_state_mgr::register_data_service_apis(nuraft_mesg::service* messaging)
     return messaging->bind_data_service_request(
                SEND_DATA, _group_id,
                [this](sisl::io_blob const& incoming_buf, boost::intrusive_ptr< sisl::GenericRpcData >& rpc_data) {
+                   rpc_data->set_comp_cb([this](boost::intrusive_ptr< sisl::GenericRpcData >&) { server_counter++; });
```

Comment:
I didn't find where in our code where calls set_comp_cb, how does it work in our prod case? Or you will bring up a new PR to do so?

Reply:
here is an example in the unit tests where we set it 
https://github.com/raakella1/nuraft_mesg/blob/comp_cb/src/tests/test_state_manager.cpp#L177

Reply:
Here is where it adds https://github.com/eBay/sisl/blob/master/include/sisl/grpc/generic_service.hpp#L70
The sisl changes were approved by Hari before https://github.com/eBay/sisl/pull/109